bazel/repositories.bzl

```diff
@@ -80,10 +80,21 @@ def python_deps(skip_targets):
             actual = "@jinja2_git//:jinja2",
         )
 
+def envoy_api_deps(skip_targets):
+  if 'envoy_api' not in skip_targets:
+    native.git_repository(
+        name = "envoy_api_git",
+        remote = "https://github.com/lyft/envoy-api.git",
+        commit = "959278cc35a89a4d2f1895f66a59c6b3de98d5e1",
+    )
+    native.bind(
```

Comment:
We can do them as needed. I'm really just trying to get a feel for how this will work, so let's leave the doco until later.